### DISCUSSION TITLE
How do I disable icons?

### DISCUSSION THREAD

**Discussion by eyenalxai** [LazyVim/LazyVim#981] [Q&A]
Tried doing like this:
```lua
  {
    "LazyVim/LazyVim",
    opts = {
      icons = {},
      colorscheme = "gruvbox",
    }
  }
```

The Gruvbox theme is set, but the icons are still present.

Thanks!

**Comment by folke** [comment]
That just configure the defaullt icons.

You can't disable the icons globally.

You'll have to do that for every plugin that has icons separately.

**Comment by heheda123123** [comment]
Can lazyvim provide this kind of global setting? I don’t want to use NF fonts, I don’t need icons
